src/commands/project/createWithEntrypoint.ts

```diff
@@ -0,0 +1,111 @@
+  static usage = "<%= command.id %> DEPOSIT_AMOUNT OWNER NAME EMAIL [URL] [SHA] [METADATA] [--type TYPE] [--spot BOOLEAN] [--renew BOOLEAN]";
+  static args: Arg[] = [
+    { name: "DEPOSIT_AMOUNT", description: "The amount of USDC to deposit into escrow.", required: true },
+    { name: "OWNER", description: "The owner for the new project.", required: true },
+    { name: "NAME", description: "The human readable name of the new project.", required: true },
+    { name: "EMAIL", description: "The project email for admin notifications.", required: true },
+    { name: "URL", description: "The public URL to fetch the content bundle.", default: "" },
+    { name: "SHA", description: "The SHA-256 checksum of the content bundle.", default: "" },
+    { name: "METADATA", description: "JSON metadata to attach to this project.", default: "" },
+    { name: "NODE_IDS", description: "The comma separated IDs of the nodes to reserve.", default: "[]" },
+  ];
+
+  static flags = {
+    ...TransactionCommand.flags,
+    spot: Flags.boolean({ description: "Change price in the current epoch only (nodes must not be reserved)." }),
+    renew: Flags.boolean({ description: "Reserve from the next epoch and on (can unreserve before next epoch start)" }),
+    type: Flags.string({
+      description: "Project type (static or nextjs)",
+      options: ["static", "nextjs"],
+      default: "static",
+    }),
+  };
+
+  public async run(): Promise<unknown> {
+    const { args, flags } = await this.parse(ProjectCreateWithEntrypoint);
+    console.log("flags: ", flags);
+    const signer = await getSigner(flags.network, flags.rpc, flags.address, flags.signer, flags.key, flags.account);
+    const signerAddress = await signer.getAddress();
+    const usdc = await getContract(flags.network, flags.abi, "USDC", signer);
+    const entrypoint = await getContract(flags.network, flags.abi, "EarthfastEntrypoint", signer);
+    const projects = await getContract(flags.network, flags.abi, "EarthfastProjects", signer);
+
+    const createProjectData = {
+        owner: args.OWNER,
+        name: args.NAME,
+        email: args.EMAIL,
+        content: args.URL,
+        checksum: args.SHA && args.SHA.length > 0 ? parseHash(args.SHA) : ethers.constants.HashZero,
+        metadata: args.METADATA,
+    };
+
+    // TODO: add a try/catch block to handle the case where ids are provided or not with calls to the respective methods
+    // const nodeIds = parseHash(args.NODE_IDS);
+    // FIXME: cleanup nodes reservations and/or add support for alternative method call
+    // if nodeIds is an empty array, reserve 2 nodes
+    const nodeIdArray = args.NODE_IDS.split(",");
+    console.log(nodeIdArray);
+    // const nodeIds = nodeIdArray.length > 0 ? nodeIdArray.map((id: string) => parseHash(id)) : [];
+    // const nodesToReserve = nodeIds.length > 0 ? nodeIds : 2;
+    const nodesToReserve = 2;
+    const slot = { last: !!flags.spot, next: !!flags.renew };
```

Comment:
Let's default these to 1 node and spot and renew to true. The amount that be can in escrow should be the limiting factor